packages/components/pages/f-checkout/src/store/checkout.module.js

```diff
@@ -250,8 +250,12 @@ export default {
          */
         getNotesConfiguration: async ({ commit, state }, { url, timeout }) => {
             const { data } = await checkoutApi.getNoteConfiguration(url, timeout);
-            const isSplitNotesEnabled = !data?.customerNotes?.serviceTypes[state.serviceType].orderNoteAccepted;
-            commit(UPDATE_NOTES_CONFIGURATION, { ...data?.customerNotes?.serviceTypes, isSplitNotesEnabled });
+            if (data?.customerNotes.serviceTypes) {
+                // This has to be done because the dinein key is sent to us as 'dineIn` ....
+                const notesByServiceType = Object.fromEntries(Object.entries(data.customerNotes.serviceTypes).map(([k, v]) => [k.toLowerCase(), v]));
```

Comment:
looks good - I was wondering if this would be easier to understand if the const was named `notesByServiceTypes` (plural rather than singular) as, if I understand correctly, this is creating an object of all the service types in lowercase?

Certainly not a big deal, approving regardless :)